packages/server/lib/errors.coffee

```diff
@@ -944,6 +944,14 @@ getMsgByType = (type, arg1 = {}, arg2, arg3) ->
 
       To avoid this error, ensure that there are no other instances of Firefox launched by Cypress running.
       """
+    when "FOLDER_NOT_ACCESSIBLE"
+      """
+      Folder #{arg1} is not writable.
+
+      Either enable write permission to this directory to enable screen shot and video collection.
+
+      Or ignore this warning if you do not need enable screen shot and video.
+      """
```

Comment:
Just updating some of the warning message to make it clear on the 'why' this is necessary plus some rewording.

```suggestion
    when "FOLDER_NOT_ACCESSIBLE"
      """
      Folder #{arg1} is not writable.
      
      Writing to this directory is required by Cypress in order to store screenshots and videos.

      Enable write permissions to this directory to ensure screenshots and videos are stored. 
      
      If you don't require screenshots or videos to be stored you can safely ignore this warning.
      """
```